File: json_to_flutter/functions/insert/insert_out.dart
import '../../models/index.dart';
import 'insert.dart';
import '../costBenefit.dart';

insertOutside(No father, Widget widget) {
  CostBenefit cost = costBenefit(father, widget);
  No putOn =
      cost.bestPosition == 0 ? father : father.children[cost.bestPosition - 1];
  insert(putOn, widget, cost.type);
}

File: src/app/jsonToFlutter/functions/insert/insert_out.js
const { costBenefit } = require("../costBenefit");
const { insertWidgetAtNoWithType } = require("./insert");

function insertOutside(father, widget) {
    let cost = costBenefit(father, widget);
    let putOn =
        cost.bestPosition == 0 ? father : father.children[cost.bestPosition - 1];
    insertWidgetAtNoWithType(widget, putOn, cost.type);
}

module.exports = { insertOutside };